src/core/render/tpl.js

```diff
@@ -37,12 +37,12 @@ export function main(config) {
   const name = config.name ? config.name : '';
 
   const aside = /* html */ `
-    <button class="sidebar-toggle" aria-label="Menu">
-      <div class="sidebar-toggle-button">
+    <button id="__sidebar-toggle" class="sidebar-toggle" aria-label="Toggle menu" aria-controls="__sidebar" aria-haspopup="true" aria-expanded="">
```

Comment:
```suggestion
    <button id="__sidebar-toggle" class="sidebar-toggle" aria-label="Toggle menu" aria-controls="__sidebar" aria-haspopup="true" aria-expanded>
```

Reply:
Better yet, I will remove the `aria-attribute` altogether and add comment explaining that this value will be set dynamically.

Per [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded):

> When a [menu](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role) is displayed, the button object that toggles the visibility of that menu has aria-expanded="true" set. When the menu is hidden, aria-expanded can be omitted. If specified when the menu is hidden, it should be set as aria-expanded="false". When a child menu is not visible, its parent [menuitem](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role) has aria-expanded. It should be set to true when the child menu is visible.

FWIW, I opted for `<button aria-expanded="">` in the source to indicate "no value" by default. The issue I have with `<button aria-expanded>` without a quotes (even if empty) is that those unfamiliar with this attribute will likely assume its existence without a value has meaning or indicates `true`, but this is incorrect.

Regardless, we can follow MDN's recommendation and still indicate that the `aria-expanded` attribute is applied dynamically to the `<button>` element via a comment.

Reply:
Placeholder `aria-expanded` placeholder has been removed.